aea/runner.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+"""This module contains the implementation of AEA multiple instances runner."""
+
+from asyncio.events import AbstractEventLoop
+from typing import Awaitable, Dict, Sequence, Type, cast
+
+from aea.aea import AEA
+from aea.helpers.multiple_executor import (
+    AbstractExecutorTask,
+    AbstractMultipleExecutor,
+    AbstractMultipleRunner,
+    AsyncExecutor,
+    ExecutorExceptionPolicies,
+    ThreadExecutor,
+)
+from aea.runtime import AsyncRuntime
+
+
+class AEAInstanceTask(AbstractExecutorTask):
+    """Task to run agent instance."""
+
+    def __init__(self, agent: AEA):
+        """
+        Init aea instance task.
+
+        :param agent: AEA instance to run within task.
+        """
+        self._agent = agent
```

Comment:
let's call an aea an aea, not an agent (as we have a different class there in the code)